src/Workspaces/Core/Portable/Workspace/Host/Status/WorkspaceStatusService.cs

```diff
@@ -13,6 +14,15 @@ internal sealed class WorkspaceStatusService : IWorkspaceStatusService
     {
         public readonly static WorkspaceStatusService Default = new WorkspaceStatusService();
 
+#pragma warning disable 0067
+        public event EventHandler<bool> StatusChanged
+        {
+            // it never used
```

Comment:
:memo: This comment is unnecessary. An explicitly implemented event with empty `add` and `remove` accessors already implies this. If the comment is not removed, it may become incorrect in the future.

```suggestion
```

Reply:
sure